docs/metrics.md

```diff
@@ -253,7 +253,7 @@ Returns the total number of virtual machine disks restored from the source virtu
 Returns the amount of space in bytes restored from the source virtual machine. Type: Gauge.
 
 ### kubevirt_vmsnapshot_persistentvolumeclaim_labels
-Returns the labels of the persistent volume claims that are used for restoring virtual machines. Type: Info.
+Returns the labels of the persistent volume claims that are used for restoring virtual machines. Type: Gauge.
```

Comment:
Is this change related to the refactoring or is it a bug you've fixed?

Reply:
50/50, metric type Info does not exist: https://prometheus.io/docs/concepts/metric_types/

and therefore the package does not provide it